<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassengerWindow</class>
 <widget class="QMainWindow" name="PassengerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>乘客端</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="topWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="userInfoLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>乘客：张三</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="leftWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="rideRequestGroup">
           <property name="title">
            <string>叫车功能</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>起点：</string>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="startLocationEdit">
              <property name="text">
               <string>0,0</string>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="minimumHeight">
               <number>35</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>终点：</string>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="endLocationEdit">
              <property name="text">
               <string>0,0</string>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="minimumHeight">
               <number>35</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>车型：</string>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="carTypeCombo">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="minimumHeight">
               <number>35</number>
              </property>
              <item>
               <property name="text">
                <string>快车</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>专车</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="requestRideBtn">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>叫车</string>
                </property>
                <property name="minimumHeight">
                 <number>45</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="cancelOrderBtn">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>取消订单</string>
                </property>
                <property name="minimumHeight">
                 <number>45</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="estimatedInfoGroup">
           <property name="title">
            <string>预估信息</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="distanceLabel">
                <property name="text">
                 <string>距离：0.0 km</string>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="timeLabel">
                <property name="text">
                 <string>时长：0.0 分钟</string>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="costLabel">
              <property name="text">
               <string>费用：¥0.00</string>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="rightWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="mapGroup">
           <property name="title">
            <string>地图</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QGraphicsView" name="mapView">
              <property name="minimumSize">
               <size>
                <width>700</width>
                <height>500</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="orderStatusGroup">
           <property name="title">
            <string>订单状态</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="orderIdLabel">
              <property name="text">
               <string>订单编号：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="statusLabel">
              <property name="text">
               <string>状态：待派单</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="tripProgressBar">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
